pkg/storage/bloom/v1/bounds.go

```diff
@@ -33,8 +64,16 @@ func (b FingerprintBounds) Hash(h hash.Hash32) error {
 	return errors.Wrap(err, "writing OwnershipRange")
 }
 
+// Addr returns the string representation of the fingerprint bounds for use in
+// content addressable storage.
+// TODO(owen-d): incorporate this into the schema so we can change it,
+// similar to `{,Parse}ExternalKey`
+func (b FingerprintBounds) Addr() string {
```

Comment:
I think naming this method `String` is more idiomatic plus allows you to do formating with %s right away.
https://goplay.tools/snippet/lmt7fpPGa7R